src/main/java/com/mongodb/kafka/connect/source/MongoSourceTask.java

```diff
@@ -217,6 +227,40 @@ public void start(final Map<String, String> props) {
     return null;
   }
 
+  private Optional<String> convertToJson(Optional<BsonDocument> jsonDocumentraw, JsonWriterSettings jsonOptions) {
```

Comment:
Other methods in this file declare their parameters as `final`. I think you can do the same.

Reply:
good catch i'm doing it